include/logging/log_msg2.h

```diff
@@ -125,7 +125,7 @@ union log_msg2_generic {
  */
 enum z_log_msg2_mode {
 	/* Runtime mode is least efficient but supports all cases thus it is
-	 * threated as a fallback method when others cannot be used.
+	 * threaded as a fallback method when others cannot be used.
```

Comment:
Shouldn't this be "treated" and not "threaded"?

Reply:
:+1:

Reply:
Thanks, I will change that and to a squash commit